spec/support/shared_context_authenticated_user.rb

```diff
@@ -0,0 +1,7 @@
+shared_context 'Authenticated User' do
+  let(:user) { create(:user) }
```

Comment:
:smile: I prefer to name it current_user instead of user, but it's ok
```suggestion
  let(:current_user) { create(:user) }
```